lms/djangoapps/ora_staff_grader/views.py

```diff
@@ -137,28 +142,25 @@ def get_submission_info(self, request, usage_id, submission_uuid):
         """
         Get submission content from ORA 'get_submission_info' XBlock.json_handler
         """
-        data = {
-            'submission_uuid': submission_uuid
-        }
-        return call_xblock_json_handler(request, usage_id, 'get_submission_info', data)
+        data = {'submission_uuid': submission_uuid}
+        response = call_xblock_json_handler(request, usage_id, 'get_submission_info', data)
+        return json.loads(response.content)
 
     def get_assessment_info(self, request, usage_id, submission_uuid):
         """
         Get assessment data from ORA 'get_assessment_info' XBlock.json_handler
         """
-        data = {
-            'submission_uuid': submission_uuid
-        }
-        return call_xblock_json_handler(request, usage_id, 'get_assessment_info', data)
+        data = {'submission_uuid': submission_uuid}
+        response = call_xblock_json_handler(request, usage_id, 'get_assessment_info', data)
+        return json.loads(response.content)
 
     def check_submission_lock(self, request, usage_id, submission_uuid):
         """
         Look up lock info for the given submission by calling the ORA's 'check_submission_lock' XBlock.json_handler
         """
-        data = {
-            'submission_uuid': submission_uuid
-        }
-        return call_xblock_json_handler(request, usage_id, 'check_submission_lock', data)
+        data = {'submission_uuid': submission_uuid}
```

Comment:
Are we doing a related PR in ORA to change this param to submission_uuid? I think it's submission_id right now

Reply:
Yes: https://github.com/edx/edx-ora2/pull/1761